CRM Solution Manager/CRM Solution/RepositoryConfigurationConstants.cs

```diff
@@ -137,7 +156,56 @@ public static string BranchName
         {
             get
             {
-                return ConfigurationManager.AppSettings["BranchName"];
+                return branchName;
+            }
+            set { }
+        }
+
+        /// <summary>
+        /// Method sets repository configuration constant property values
+        /// </summary>
+        /// <param name="retrievedConfigurationSettingsList">entity collection</param>
+        public override void SetRepositoryConfigurationProperties(EntityCollection retrievedConfigurationSettingsList)
+        {
+            foreach (Entity setting in retrievedConfigurationSettingsList.Entities)
+            {
+                string key = setting.GetAttributeValue<string>("syed_name");
+
+                switch (key)
+                {
+                    case "RepositorySolutionFolder":
```

Comment:
move all these keys "RepositorySolutionFolder" and other keys from string hardcodings to a constants files

Reply:
do this changes everywhere

Reply:
done.